packages/grid/x-grid-data-generator/src/commodities.columns.tsx

```diff
@@ -222,6 +234,8 @@ export const getCommodityColumns: (editable?: boolean) => any[] = (editable = fa
       headerName: 'Counterparty Country',
       generateData: randomCountry,
       renderCell: renderCountry,
+      type: 'singleSelect',
+      valueOptions: COUNTRY_ISO_OPTIONS,
```

Comment:
Would it make sense?

```suggestion
      valueOptions: COUNTRY_ISO_OPTIONS,
      editable: true,
```

Reply:
Am that one is editable (I tested it before)

Reply:
I don't understand. Are we looking at the same code?